pkg/lockfile/match-package-json.go

```diff
@@ -1,17 +1,24 @@
 package lockfile
 
 import (
-	"fmt"
 	"io"
 	"strings"
 
 	"github.com/google/osv-scanner/internal/cachedregexp"
+
 	"github.com/google/osv-scanner/internal/utility/fileposition"
 	"github.com/google/osv-scanner/pkg/models"
 )
 
 type PackageJSONMatcher struct{}
 
+const (
+	namePrefix    = "\""
+	nameSuffix    = "\":"
+	versionPrefix = ":\\s*\""
+	versionSuffix = "\",?"
+)
+
 func (m PackageJSONMatcher) GetSourceFile(lockfile DepFile) (DepFile, error) {
```

Comment:
This method starts to be complex to read, can we try to split it in multiple private methods ?

Reply:
`Match` method split in multiple smaller private methods. Feel free to suggest any other way of approaching this 👍